modules/core/src/models/claim.ts

```diff
@@ -167,3 +170,23 @@ export enum SharedProfileValueResolvingMethod {
     FROM_SHARED_PROFILE = "FromSharedProfile",
     FROM_FIRST_FOUND_IN_HIERARCHY = "FromFirstFoundInHierarchy"
 }
+
+/**
+ * Enum representing the data types of claims.
+ * - STRING: A string value.
+ * - INTEGER: An integer value.
+ * - DECIMAL: A decimal value.
+ * - BOOLEAN: A boolean value.
+ * - DATE_TIME: A date and time value.
+ * - COMPLEX: A complex value.
+ */
+export enum DataType {
+    STRING = "string",
```

Comment:
Wondering what is the difference between STRING and TEXT data types. Cannot we only keep one of those?

Reply:
For the backend only STRING data type is used.

But for the frontend,
if dataType=STRING and canonicalValues is empty it is TEXT data type.
if dataType=STRING and canonicalValues is not empty it is OPTIONS data type.